Modules/_testcapi/object.c

```diff
@@ -131,13 +131,22 @@ pyobject_enable_deferred_refcount(PyObject *self, PyObject *obj)
     return PyLong_FromLong(result);
 }
 
+
+static PyObject *
+is_immortal(PyObject *self, PyObject *op)
+{
+    return PyBool_FromLong(PyUnstable_IsImmortal(op));
```

Comment:
```suggestion
    NULLABLE(op)
    return PyLong_FromLong(PyUnstable_IsImmortal(op));
```
This allows to test with NULL, and check if the result is not 0 or 1.

Reply:
I think it makes sense to crash with `NULL`.